src/components/JudgingCriteria.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+
+export default class JudgingCriteria extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            criteria: [
+                { category: 'Usefulness', desc: "fits our profile", points: "35%" },
+                { category: 'Creativity', desc: "out of the box", points: "25%" },
+                { category: 'Technical Complexity', desc: "complex calculation, computing", points: "20%" },
+                { category: 'Overall Quality', desc:"progress made during the hackathon", points: "15%" },
+                { category: 'Presentation', desc: "clear process description/demo", points: "5%" }
+            ]
+        }
+    }
+    renderTableData() {
+        return this.state.criteria.map((criterion, index) => {
+            const { category, desc, points } = criterion //destructuring
+            return (
+                <tr key={category}  >
```

Comment:
everything looks really awesome! I would just suggest using Bulma for css stuff 😉But you've done all the hard work!!
https://bulma.io/documentation/elements/table/